packages/app/src/sandbox/eval/loaders/@babel-runtime.no-webpack.js

```diff
@@ -421,7 +421,7 @@ function M(e) {
     );
   var o = e.descriptor;
   K(e, 'elements', 'An element descriptor');
-  var i = { kind: r, key: t, placement: n, descriptor: Object.assign({}, o) };
```

Comment:
Can you please allso revert this file?

This is generated

Reply:
Reverted 🙂